database/migrations/2023_09_06_072008_create_users_table.php

```diff
@@ -15,8 +15,9 @@ public function up(): void
             $table->id();
             $table->string('name');
             $table->string('email')->unique();
+            $table->string('provider_id')->nullable();
             $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
+            $table->string('password')->nullable();
```

Comment:
the `password` field has been removed from the `User` model,
for the database table to be compatible with the models, we have to:
or not removing password field from the model or remove it from here too